jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -399,6 +399,9 @@ def _append_angle_correctly(self, flux_unit, angle_unit):
             sb_unit_selected = flux_unit + ' / ' + angle_unit
 
         if sb_unit_selected:
-            self.sb_unit_selected = sb_unit_selected
+            # convert string to and from u.Unit to get rid of any
+            # formatting inconstancies, order of units in string
+            # for a composite unit matters
+            sb_unit_selected = u.Unit(sb_unit_selected).to_string()
```

Comment:
Should the self be kept?
```suggestion
            self.sb_unit_selected = u.Unit(sb_unit_selected).to_string()
```

Reply:
I intentionally removed setting it here. Its set from the return value of this function on line 222, and the other times its called it doesn't need to set the attribute again